README.md

```diff
@@ -0,0 +1,74 @@
+# `search` formula
+
+This repository contains instructions for installing and configuring the `search` project.
+
+This repository should be structured as any Saltstack formula should, but it should also conform to the structure 
+required by the [builder](https://github.com/elifesciences/builder) project.
+
+## creating and restoring Elasticsearch snapshots
+
+Creating a snapshot of all indices in an ElasticSearch (ES) database will require a 'repository' and approximately the 
+same amount of disk space as the set of indices themselves.
+
+An ES 'repository' is a formal location shared across all nodes in an ES cluster.
+
+A snapshot is created in a repository and is available across all nodes. If there is a single node then the repository 
+isn't that special, it's just a directory on the filesystem.
+
+Creating repositories and snapshots is easy. Subsequent snapshots in the same repository are incremental and will only 
+capture the changes made since the last snapshot.
+
+### creating repositories
+
+Before a snapshot can exist it must be created in a place ES knows about called a 'repository'. `fs` type repositories 
+exist on the filesystem and ES needs to know their location as well with the setting `path.repo` in 
+`/etc/elasticsearch/elasticsearch.yml`.
+
+To create a repository issue this api call: `PUT /_snapshot/{repo}` with the payload:
+
+```json
+{"type": "fs", "settings": {"location": "$repo"}}
+```
+
+Where `$repo` is the name of a path rooted in the `path.repo` setting (multiple paths may exist).
+
+[Reference](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/modules-snapshots.html#_repositories)
```

Comment:
I have a feeling we are on an outdated version, unrelated

Reply:
(we are ;)

but for now the PPA works and after the 18.04 upgrade, if we're game, we can look at upgrading to ES 7